src/main/java/frc/robot/subsystems/snapback/shooter/ShooterIO.java

```diff
@@ -17,12 +17,16 @@ public static class ShooterIOInputs {
     public double leftAppliedVolts;
     public double leftCurrentAmps;
     public double leftTemperatureCelsius;
+    public double leftVelocitySetpointRadiansPerSecond;
+    public double leftVelocityErrorRadiansPerSecond;
 
     public Rotation2d rightPosition;
     public double rightVelocityRadiansPerSecond;
     public double rightAppliedVolts;
     public double rightCurrentAmps;
     public double rightTemperatureCelsius;
```

Comment:
Need to add input for right goal velocity

```suggestion
    public double rightVelocityGoalRadiansPerSecond;
```